### PR TITLE
support rhel based runner

### PR DESCRIPTION
**Pull request by rootfs**
Tested with RHEL 9 AMI `ami-078cbc4c2d057c244` with t2.micro instance

### CODE REVIEW

File: entrypoint.sh
Comment (SamYuan1990) [inline entrypoint.sh:117]: do we need yum update here?

File: entrypoint.sh
Comment (SamYuan1990) [inline entrypoint.sh:156]: 2 >&1 will change log output stream, is this change save enough?

File: entrypoint.sh
Comment (SamYuan1990) [inline entrypoint.sh:215]: from code spec we can have this exception catch, but should we move this code comments into FAQ as a part of readme?
I am not sure if there any user/use case want to break if the price is ... higher than 0.0035200000000000023 ?

**Review by SamYuan1990 — CHANGES_REQUESTED** [review]
btw, do we need add any test case related with this PR?

File: entrypoint.sh
Comment (rootfs) [reply entrypoint.sh:117]: no, this command can install the libraries

File: entrypoint.sh
Comment (rootfs) [reply entrypoint.sh:156]: we need to pick up the error message and parse the min bid price from the output, `2&1` can get the output

File: entrypoint.sh
Comment (rootfs) [reply entrypoint.sh:215]: the two way of adjusting bid prices are here.  If we have more heuristics, I feel then we should factor this code and support a more generic algorithm.

**Comment by rootfs** [comment]
> btw, do we need add any test case related with this PR?

can you refactor the CI after this PR is merged?

**Comment by SamYuan1990** [comment]
> > btw, do we need add any test case related with this PR?
> 
> can you refactor the CI after this PR is merged?

We need document the metrics before refactor the CI.
IMO, it better to have a github issue for this.
1. what are the parameters?
2. basing on those parameters make the CI parameter able.
3. make a minimum scope of CI to cover the cases, and as input for step 2.
4. create badge for each case and update on readme file.

so that when user, for example kepler model server to consume this repo to create model on aws, the maintainer from kepler model server just need to:
1. following our steps to set up aws security group and iam, etc....
2. select a sub scope according our tested metric.
3. replace our test on self-hosted instance with their own logic.

File: entrypoint.sh
Comment (SamYuan1990) [reply entrypoint.sh:215]: new issue created accordingly/

**Review by SamYuan1990 — APPROVED** [review]

**Pull request merged**
